### PR TITLE
Use `assertNull(...)` instead of `assertEquals(null, ...)`

### PR DESCRIPTION
**Pull request by alxndrsn**

### CODE REVIEW

File: collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java [hunk 1]
```diff
@@ -70,8 +71,8 @@ public void resetSettingsTest() throws IOException {
         resetAppState(Collections.singletonList(ResetUtility.ResetAction.RESET_PREFERENCES));
 
         SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(Collect.getInstance());
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_USERNAME, null));
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_PASSWORD, null));
+        assertNull(settings.getString(PreferenceKeys.KEY_USERNAME, null));
```
Comment (lognaturel) [inline collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: I don't think this needs `null` anymore?

Comment (alxndrsn) [reply collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: I don't think there's a 1-arg option for that method: https://developer.android.com/reference/android/content/SharedPreferences.html#getString(java.lang.String,%20java.lang.String)

Comment (alxndrsn) [reply collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: ...and has github swallowed a line here?

Comment (lognaturel) [reply collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: I'm sorry, that was my brain playing tricks on me. I thought that was an `assertNull` argument that made no sense.

Comment (alxndrsn) [reply collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: Ah, it's definitely included in the _Files changed_ tab :~)

Comment (lognaturel) [reply collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java:74]: Must be time for brain food. 🍜

**Pull request merged**